src/DayCounts.jl

```diff
@@ -113,35 +113,36 @@ This ensures that:
  - [EMU and Market Conventions: Recent Developments, ISDA - BS:9951.1](https://www.isda.org/a/AIJEE/1998-ISDA-memo-EMU-and-Market-Conventions-Recent-Developments.pdf), §4. The Actual/Actual Day Count Convention.
 """
 struct ActualActualICMA <: DayCount
-    schedule::StepRange{Date,Month}
+    frequency
+    schedule::Vector{Date}
```

Comment:
Maybe make it parametric (e.g. `T` where `T<:AbstractVector{Date}` so we can use `StepRange` objects directly?

Reply:
Sure. Will do that 👍